### ISSUE TITLE
Systemjs 0.20 

### ISSUE THREAD

**Issue by salfield** [systemjs/systemjs#1570]
When using certain plugins with Systemjs 0.20+ and babel-loader  (e.g. "babel-plugin-transform-decorators-legacy") I get the following error:

> Error: Plugin 1 specified in "base" provided an invalid property of "default"

I guess this is due to the fact that the plugin exports "default" and the import behaviour "Removes the ability to use named imports from non-ES modules", as per the release notes. Assuming I'm right, what is the method to get the plugin running? In the meantime I'll stick with systemjs 0.19. 

I'm actually using jspm but I think this is a systemjs issue (or even babel-loader) issue.  Here are my babelOptions which are sufficient to throw the error.

 ```
 babelOptions: {
    "presets": [
      "babel-preset-stage-1"
    ],
    "plugins": [
      "babel-plugin-transform-react-jsx",
      "babel-plugin-transform-decorators-legacy"
    ]
  },
```

**Comment by Rush** [comment]
@salfield if you could prepare a standalone test case it would probably be solved faster. I actually encountered the same issue today but then it disappeared after changing some configs/plugins.

**Comment by salfield** [comment]
@Rush sure, well here it is:

https://github.com/salfield/systemjs_plugin_issue

Just run 

> npm install .

 and then

> jspm install

Start a web server on that root of the directory and check the errors in the console.

**Comment by guybedford** [comment]
This looks like a duplicate of https://github.com/ModuleLoader/es-module-loader/issues/526.

**Comment by guybedford** [comment]
This introp should be fixed up in the 0.0.21 release of the Babel plugin.
